mlflow/sklearn-triton/wine/odahuflow/packaging.odahuflow.yaml

```diff
@@ -0,0 +1,12 @@
+id: wine
+kind: ModelPackaging
+spec:
+  artifactName: "<fill-in>"
+  integrationName: docker-rest
```

Comment:
Here should be the `docker-triton` packager, I suppose. You want to leave it as is until we complete the full Triton pipeline?

Reply:
Sure. Fixed.